src/views/Entries/QualityControl/EntryCard/index.tsx

```diff
@@ -287,90 +297,116 @@ function EntryCard(props: EntryCardProps) {
                         />
                     </div>
                 </section>
-                <section className={styles.bottom}>
-                    <div className={styles.row}>
-                        <div className={styles.source}>
-                            { leadSource && (
-                                <Icon
-                                    name="world"
-                                />
-                            )}
+                {isVisible &&
```

Comment:
Use brackets to group jsx blocks
```suggestion
                {isVisible && (
```